### PR TITLE
Randomly generated stat description for oddities

### PR DESCRIPTION
**Pull request by Alliostra**
## About The Pull Request

The descriptions that tell you what stats you gain from resting with an oddity will now be randomly generated. This ~~sometimes~~, who am I kidding, often leads to phrases that would make our former great leader blush. But hey, it only adds to the charm and atmosphere, if items that don't make any sense also contain engrish that doesn't make any sense.
grafik

## Why It's Good For The Game

Hopefully makes especially newer people go less "; selling high COG low ROB w/ fast walker, buying GF".
Also funny reerenglish

## Changelog
:cl:
add: The descriptions of oddity stats are now randomly generated.
/:cl:

### CODE REVIEW

**Comment by Firefox13** [comment]
I don't really like the idea of shitty english for the sake of shitty english

**Comment by MemedHams** [comment]
It's funny the first time you see it, but it's borderline-illegibility holds it back from being really charming. You might want to consider organizing it into basic blocs of different word types, like the artist statue terms.

**Comment by MemedHams** [comment]
Honestly, I might temporarily delay the artist changes and try my hand at this as well

**Comment by Alliostra** [comment]
Well, If the maintainers don't mind I'd probably just hardcode in a bunch of grammatically correct phrases since that's going to be much less work.

**Comment by TheShown911** [comment]
I mind.

File: code/modules/sanity/inspiration_component.dm [hunk 1]
```diff
@@ -33,18 +33,75 @@
...
+				stat_color = "<span class='blue'>"
+				if(aspect_noun_or_adjective == "noun")
+					aspect = pick(nouns_weak)
+				else
+					aspect = pick(adjectives_weak)
 			else
 				continue
-		to_chat(user, SPAN_NOTICE("This item has [aspect] aspect of [stat]"))
+		if(aspect_noun_or_adjective == "noun")
+			message = pick("This item has [a_or_an] [stat_color][aspect]</span> aspect of [stat_noun]", "[a_or_an] [stat_color][aspect]</span> Aura of [stat_noun] pertains to it")
```
Comment (Trilbyspaceclone) [inline code/modules/sanity/inspiration_component.dm:100]:
```suggestion
			message = pick("This item has [a_or_an] [stat_color][aspect]</span> aspect of [stat_noun]", "Aura of [stat_color][aspect]</span> [stat_noun] pertains to it")
```
So we dont start the message with lower cased a or an
and then dont have Aura capsized in the middle

**Comment by Alliostra** [comment]
Closing this for now until I can figure out a better way

**Pull request closed without merging**